include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -83,6 +83,8 @@ ERROR(error_unknown_arg,none,
   "unknown argument: '%0'", (StringRef))
 ERROR(error_invalid_arg_value,none,
   "invalid value '%1' in '%0'", (StringRef, StringRef))
+NOTE(note_cannot_multithread_batch_mode,none,
+  "ignoring -num-threads argument, cannot multithread batch mode", ())
```

Comment:
Yes, it *must* be a warning. It's invalid to emit a note on its own.

(Arguably DiagnosticEngine would enforce this, but the convenience of "just emit the warning and then follow it with a note" has kept us from doing so so far.)

Reply:
Tiny nitpick: ";" instead of ","